### ISSUE TITLE
SmartCity-Accessibility #40 failed

### ISSUE THREAD

**Issue by JuanJenkinsBot** [TechnionYP5777/SmartCity-Accessibility#84]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 57 lines...]


[INFO] 
[INFO] --- maven-clean-plugin:2.5:clean (default-clean) @ SmartCity-Accessibility ---
[INFO] 
[INFO] --- maven-resources-plugin:2.6:resources (default-resources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:compile (default-compile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 41 source files to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/classes
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java: Some input files use or override a deprecated API.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/main/java/smartcity/accessibility/search/SearchQuery.java: Recompile with -Xlint:deprecation for details.
[INFO] 
[INFO] --- maven-resources-plugin:2.6:testResources (default-testResources) @ SmartCity-Accessibility ---
[INFO] Using 'UTF-8' encoding to copy filtered resources.
[INFO] skip non existing resourceDirectory /var/lib/jenkins/workspace/SmartCity-Accessibility/src/test/resources
[INFO] 
[INFO] --- maven-compiler-plugin:3.5.1:testCompile (default-testCompile) @ SmartCity-Accessibility ---
[INFO] Changes detected - recompiling the module!
[INFO] Compiling 6 source files to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/test-classes
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/test/java/smartcity/accessibility/search/SearchTest.java: /var/lib/jenkins/workspace/SmartCity-Accessibility/src/test/java/smartcity/accessibility/search/SearchTest.java uses or overrides a deprecated API.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/test/java/smartcity/accessibility/search/SearchTest.java: Recompile with -Xlint:deprecation for details.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/test/java/smartcity/accessibility/socialnetwork/AdminTest.java: /var/lib/jenkins/workspace/SmartCity-Accessibility/src/test/java/smartcity/accessibility/socialnetwork/AdminTest.java uses unchecked or unsafe operations.
[INFO] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/test/java/smartcity/accessibility/socialnetwork/AdminTest.java: Recompile with -Xlint:unchecked for details.
[INFO] -------------------------------------------------------------
[ERROR] COMPILATION ERROR : 
[INFO] -------------------------------------------------------------
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/test/java/smartcity/accessibility/socialnetwork/BrokenAdminTest.java:[27,25] method does not override or implement a method from a supertype
[INFO] 1 error
[INFO] -------------------------------------------------------------
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 13.367 s
[INFO] Finished at: 2016-12-19T19:59:15+00:00
[INFO] Final Memory: 28M/192M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.5.1:testCompile (default-testCompile) on project SmartCity-Accessibility: Compilation failure
[ERROR] /var/lib/jenkins/workspace/SmartCity-Accessibility/src/test/java/smartcity/accessibility/socialnetwork/BrokenAdminTest.java:[27,25] method does not override or implement a method from a supertype
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:

[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/40/)

**Comment by JuanJenkinsBot** [comment]
Build 'SmartCity-Accessibility' is failing!

Last 50 lines of build output:

```
[...truncated 2900 lines...]
[40.79467,-105.908096]
[40.794536,-105.908828]
[40.794116,-105.909484]
[40.794082,-105.909767]
[40.794158,-105.910125]
[40.794952,-105.911979]
[40.797584,-105.917884]
[40.798011,-105.9197]
[40.798042,-105.920211]
[40.797946,-105.920875]
[40.797683,-105.921836]
[40.797634,-105.922073]
[40.7975,-105.922744]
[40.797012,-105.924072]
[40.79697,-105.924331]
[40.79726,-105.925544]
[40.797401,-105.926605]
[40.797573,-105.927276]
[40.79779,-105.927886]
[40.797939,-105.928184]
[40.79816,-105.928405]
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 4.8 sec - in smartcity.accessibility.navigation.NavigationTest

Results :

Tests in error: 
  UserManagerTest.test2:63 » v Failed to get Browser browserChannel 0
  BrokenAdminTest.setUp:21 » v Failed to get Browser browserChannel 1

Tests run: 14, Failures: 0, Errors: 2, Skipped: 1

[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 25:49 min
[INFO] Finished at: 2016-12-19T20:26:36+00:00
[INFO] Final Memory: 24M/166M
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-surefire-plugin:2.17:test (default-test) on project SmartCity-Accessibility: There are test failures.
[ERROR] 
[ERROR] Please refer to /var/lib/jenkins/workspace/SmartCity-Accessibility/target/surefire-reports for the individual test results.
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
Build step 'Invoke top-level Maven targets' marked build as failure

```

Changes since last successful build:
- [Alexander Kaplan] 45f37cbb49f9ed62413aff86761e1638eb440b0b - compile fix



[View full output](http://163.172.171.65:8080/job/SmartCity-Accessibility/41/)

**Comment by JuanJenkinsBot** [comment]
Build was fixed!
